public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade

```diff
@@ -449,14 +443,14 @@ table(width="100%")
         It displays an element whose `*ngSwitchCase` matches the current `ngSwitch` expression value.
 
         In this example, if `favoriteHero` is not set, the `ngSwitch` value is `null`
-        and we see the `*ngSwitchDefault` paragraph, "Please enter ...".
-        If the `favoriteHero` is set, it checks the movie hero by calling a component method.
-        If that method returns `true`, we see "Excellent choice!".
-        If that methods returns `false`, we see "No movie, sorry!".
+        and `*ngSwitchDefault` displays, "Please enter ...".
+        If `favoriteHero` is set, it checks the movie hero by calling a component method.
+        If that method returns `true`, it displays: "Excellent choice!"
```

Comment:
E.g., this sound like "it" refers to "that method". I think "it" is actually "the app".